File: KarateKlub/KarateKlub.Api/Models/NeizmireneClanarinePregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class NeizmireneClanarinePregledVM
    {
        public class Row
        {
            public int stavkaId;
            public int clanarinaId;
            public int clanKlubaId;
            public string clanKluba;
            public DateTime datumOd;
            public DateTime datumDo;
        }
        public List<Row> rows;
        }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_blagajnik/model/NeizmireneClanarinePregledVM.java
package com.example.x.karateklub.area_blagajnik.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class NeizmireneClanarinePregledVM implements Serializable {
    public static class Row implements Serializable{

        public int stavkaId;
        public int clanarinaId;
        public int clanKlubaId;
        public String clanKluba;
        public Date datumOd;
        public Date datumDo;
    }
public List<Row> rows;
}
